### DISCUSSION TITLE
Is link about animating icons in admonitions right?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2360] [Q&A] [answered]
There is a link in the [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) page about the posibility of animating the icon used.
Clicking on it however redirects you to the Python Markdown Extension "Details". Is this intentional?

My guess is, that it should actually link to the [animation section of icons-emojis.md](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#with-animations).
If that is the case would I have a PR ready to open, which would correct this.

**Comment by squidfunk** [comment, marked as answer]
In case of stale links, a PR is always appreciated!

**Reply by Andre601** [reply]
So is this a wrong link or not? Your response doesn't make it that clear.

**Reply by squidfunk** [reply]
I'm sorry, I haven't made this clear: yes, the link is wrong.

**Reply by jromal** [reply]
Deleted to make @Andre601 happy.

**Reply by Andre601** [reply]
This isn't the discussion to ask this in.
Please make a separate one.
